### PR TITLE
[chores] Added Github Action for CI

### PR DESCRIPTION
**Pull request by pandafy**
- Removed Travis integration

### CODE REVIEW

File: runtests.py [hunk 1]
```diff
@@ -20,4 +20,4 @@
         args.insert(2, 'openwisp2')
         app_dir = ['tests', 'openwisp2', 'sample_notifications']
     execute_from_command_line(args)
-    sys.exit(pytest.main([os.path.join(*app_dir, 'tests', 'test_websockets.py')]))
+    # sys.exit(pytest.main([os.path.join(*app_dir, 'tests', 'test_websockets.py')]))
```
Comment (atb00ker) [inline runtests.py:23]: This is commented out. :smile:

Comment (pandafy) [reply runtests.py:23]: This is has been done deliberately due to https://github.com/openwisp/openwisp-notifications/issues/145. The build will stall if I don't comment out this code which will make it hard for us to judge this.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install system packages
+      run: sudo apt -y install sqlite3 gdal-bin
```
Comment (dwang) [inline .github/workflows/build.yml:47]:
```suggestion
      run: sudo apt-get -y install sqlite3 gdal-bin
```

[`WARNING: apt does not have a stable CLI interface. Use with caution in scripts.`](https://github.com/openwisp/openwisp-notifications/runs/1364607740?check_suite_focus=true#step:5:8)

We can make it less noisy by using the `-qq` flag if you want. Also, do you think we need to run `apt-get update` before as well? It works right now but I'm not sure if something might break in the future.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+    - name: Install python dependencies dependencies
```
Comment (dwang) [inline .github/workflows/build.yml:49]:
```suggestion
    - name: Install python dependencies
```

File: runtests.py (hunk 1 above)
Comment (nemesisdesign) [reply runtests.py:23]: Now that #145 is merged, can this line be uncommented?

**Review by nemesisdesign — APPROVED** [review]
First impression is great! :+1: 

See my comment below, I think we can proceed after that.

**Pull request merged**